kod/object/active/holder/nomoveon/battler/monster.kod

```diff
@@ -4918,10 +5092,10 @@ messages:
       % avoid division by zero, if objects teleport
       % objects with viSpeed = 0 don't need seamless movements in the client
       % so we just assume it is enough to teleport them once a second
-      if viSpeed > 0
+      if Send(self,@GetSpeed) > 0
       {
```

Comment:
14. Monster  GetMoveTime    33842618
GetMoveTime is the 14th highest called message on 103. Most of the CPU usage of blakserv is due to the monster AI, which includes messages such as this one. Using C calls here instead of using the properties (which you shouldn't be modifying anyway) does add up and is likely to cause us issues in the future. The messages related to monster movement and attacking has already been optimised somewhat to increase speed/reduce chances of future issues. Editing any of those messages requires much more care than would be used elsewhere in the codebase.

Use the variable instead.